javascriptv3/example_code/sns/README.md

```diff
@@ -1,14 +1,16 @@
 # TypeScript environment for Amazon Sample Email Service (SES) examples
 
-Environment for AWS SDK for JavaScript (V3) AWS Identity and Access Management (SES) examples. For more information, see the [AWS documentation for these examples](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sns-examples.html).
+Environment for AWS SDK for JavaScript (v3) AWS Identity and Access Management (SES) examples. 
```

Comment:
Sample -> Simple
AWS Identity and Access Management -> Amazon Simple Email Service

Reply:
Done